tsconfig.json

```diff
@@ -14,6 +14,6 @@
     "isolatedModules": true,
     "jsx": "preserve"
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
```

Comment:
`**/*.ts` and `**/*.tsx` would cover all ts and tsx files
please convert `firebaseClient` and `firebaseAdmin` to typescript as well so that you can take advantages of types and avoid any type errors